packages/widgets/Source/Viewer/Viewer.js

```diff
@@ -872,25 +836,12 @@ Either specify options.terrainProvider instead or set options.baseLayerPicker to
   this._eventHelper = eventHelper;
   this._lastWidth = 0;
   this._lastHeight = 0;
-  this._allowDataSourcesToSuspendAnimation = true;
-  this._entityView = undefined;
   this._enableInfoOrSelection = defined(infoBox) || defined(selectionIndicator);
-  this._clockTrackedDataSource = undefined;
-  this._trackedEntity = undefined;
-  this._needTrackedEntityUpdate = false;
   this._selectedEntity = undefined;
-  this._zoomIsFlight = false;
-  this._zoomTarget = undefined;
-  this._zoomPromise = undefined;
-  this._zoomOptions = undefined;
   this._selectedEntityChanged = new Event();
-  this._trackedEntityChanged = new Event();
 
-  knockout.track(this, [
-    "_trackedEntity",
-    "_selectedEntity",
-    "_clockTrackedDataSource",
-  ]);
```

Comment:
Just commenting on this. I'm not completely certain why this was here previously but I don't see anything to replace it and I just want to be sure it's not setting something up that's used elsewhere that we're missing. Maybe something with the selection details widget?

Reply:
@ggetz do you know if this was used in any sandcastles or older widgets at some point?

Reply:
Not to my knowledge. Looks like this are just relics. The subscriptions themselves appear to have been removed in https://github.com/CesiumGS/cesium/commit/dfb452824b2e1b7db0e93078575aefd428c9b630#diff-f348e35101f15dcc67c86e7ec8060650fd7d17d0f3f2eeed58ffc7455520a5bbL738.